packages/echo/src/remix.ts

```diff
@@ -0,0 +1,41 @@
+import { EchoRequestHandler, ServeHandlerOptions } from './handler';
+import { type SupportedFrameworkName } from './types';
+
+export const frameworkName: SupportedFrameworkName = 'remix';
+
+export const serve = (
+  options: ServeHandlerOptions
+): ((ctx: { request: Request; context?: unknown }) => Promise<Response>) => {
+  const handler = new EchoRequestHandler({
+    frameworkName,
+    ...options,
+    handler: ({ request: req }: { request: Request }) => {
+      return {
+        body: () => req.json(),
+        headers: (key) => req.headers.get(key),
+        method: () => req.method,
+        url: () => new URL(req.url, `https://${req.headers.get('host') || ''}`),
+        transformResponse: ({ body, status, headers }): Response => {
```

Comment:
For future reference, it would be beneficial to introduce a RequestLike interface that extends the default Request and pass that instead of passing a deconstructed version of the request parts.

Reply:
100% agree. Let's add a few more frameworks and see where the dust settles.